### ISSUE TITLE
Refresh control doesn't stop spinning

### ISSUE THREAD

**Issue by tosbaha** [KyoheiG3/SpringIndicator#2]
I created tableView by code and added refresh control to the tableview. However after dragging ends refresh control doesn't stop spinning. I am using latest Xcode iOS 8.3 SDK targetting iOS 7
```swift 
class ViewController: UIViewController {

    lazy var headerView: HeaderView = {
        let view = HeaderView()
        return view
    }()
   
    lazy var tableView: UITableView = {
        let tableView = UITableView()
        tableView.delegate = self
        tableView.dataSource = self
        tableView.registerClass(StopTableViewCell.self, forCellReuseIdentifier: "cell")
        return tableView
    }()

       override func viewDidLoad() {
        super.viewDidLoad()
        view.addSubview(tableView)
        self.automaticallyAdjustsScrollViewInsets = true
        tableView.frame = view.frame
        tableView.rowHeight = UITableViewAutomaticDimension;
        tableView.estimatedRowHeight = 85
        self.edgesForExtendedLayout = UIRectEdge.None
        refreshControl.indicator.lineColor = UIColor.redColor()
        refreshControl.addTarget(self, action: "onRefresh", forControlEvents: .ValueChanged)
        self.tableView.addSubview(refreshControl)
   }

    func onRefresh() {
        for var i = 1; i < 5; i++ {
            println("Doing Something")
        }
        refreshControl.endRefreshing()
    }
extension ViewController: UITableViewDelegate, UITableViewDataSource {

    func tableView(tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
        return headerView
    }
....
```

onRefresh is called because I see debug output. However refreshControl.endRefreshing() doesn't stop the animation.

**Comment by KyoheiG3** [comment]
It's no good in the same run loop and action of `UIControl`.

Think well go with if for example, the following:

```Swift
func onRefresh() {
    for var i = 1; i < 5; i++ {
        println("Doing Something")
    }
    let delay = 3.0 * Double(NSEC_PER_SEC)
    let time  = dispatch_time(DISPATCH_TIME_NOW, Int64(delay))
    dispatch_after(time, dispatch_get_main_queue()) {
        self.refreshControl.endRefreshing()
    }
}
```

**Comment by tosbaha** [comment]
Adding a delay solved the problem. However, after the refresh table doesn't bounce back smoothly. I added animation to show the behavior. 
![glitch](https://cloud.githubusercontent.com/assets/971530/7478086/d3a995da-f360-11e4-8689-4ff1fcc8543b.gif)

**Comment by KyoheiG3** [comment]
If sectoin header does not move in a change of `contentOffset`, it's a difficult problem.
Once it possibly `UITableView` of style might be better if it was `.Grouped`.

**Comment by tosbaha** [comment]
I think it can be done smoothly. Since PZPullRefreshView library works smoothly. in the same application. The way it does is doing below line of code at the end of refreshing of operation.
```swift
  UIView.beginAnimations(nil, context: nil)
  UIView.setAnimationDuration(0.4)
  scrollView.contentInset = UIEdgeInsetsMake(0, 0, 0, 0)
  UIView.commitAnimations()
```

**Comment by KyoheiG3** [comment]
If you change the `contentInset` in Animation block, it would be animation at the time of re-use of the `Cell`. If size of the `Cell` does not change there is no problem.

**Comment by tosbaha** [comment]
Sorry I didn't get what you said. Will my fix cause a problem ? If so, what is the better way to stop that jumping issue? Thanks.

**Comment by KyoheiG3** [comment]
I will occur `UICollectionView` the problem has been implemented. You want to use it to change only there if there is no problem. I will try to investigate the jumping issue. It is better to change the `contentInset` outside of at least animation block.

**Comment by KyoheiG3** [comment]
Fixed in `1.1.2`.

I think that it becomes the same movement as `UIRefreshControl`.

**Comment by tosbaha** [comment]
Thanks your fix worked on my example. Also as you suggested before, using ```UIEdgeInsetsZero``` after animation block like below has the same effect
```swift
        completionBlock()
                            
                    }
                    scrollView.contentInset = UIEdgeInsetsZero
```
However, I think your new approach is better. Thanks for fixing it.
